src/Test/Perf/ConsumptionParser/ConsumptionParser.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Xml.Linq;
+
+namespace Roslyn.Test.Performance.Utilities.ConsumptionParser
+{
+
+    internal struct ConsumptionParse
+    {
+        public IEnumerable<ScenarioResult> Scenarios { get; }
```

Comment:
In some of your structs properties are on top, on others they are on the bottom.  We tend to default to on top in Roslyn.